manifest.aspnet.json

```diff
@@ -0,0 +1,186 @@
+{
+  "readmePath": "README.md",
```

Comment:
This should not be declaring the product family readme.  It is already declared in the [manifest.json file](https://github.com/microsoft/dotnet-framework-docker/blob/master/manifest.json#L2)